src/components/opinion/cutout.tsx

```diff
@@ -31,24 +24,27 @@ const ImageStyles = css`
 
 interface Props {
     contributors: Contributor[];
-    imageMappings: ImageMappings;
     className: SerializedStyles;
 }
 
-const Cutout = ({ contributors, imageMappings, className }: Props): JSX.Element | null => {
+const Cutout = ({ contributors, className }: Props): JSX.Element | null => {
     const [contributor] = contributors;
 
-    if (isSingleContributor(contributors) && contributor.bylineLargeImageUrl) {
-        const imgSrc = src(imageMappings, contributor.bylineLargeImageUrl, imageWidth*3);
-        return (
-            <div css={[className, Styles]}>
-                <img css={ImageStyles} src={imgSrc} alt={contributor.webTitle}/>
-            </div>
-        );
+    if (!isSingleContributor(contributors)) {
```

Comment:
These logic is duplicated in a few places, maybe a future cleanup

Reply:
Yeah good point.